lib/irb/command/disable_irb.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module IRB
+  # :stopdoc:
+
+  module Command
+    class DisableIrb < Base
+      category "IRB"
+      description "Disable binding.irb."
+
+      def execute(*)
+        ::Binding.define_method(:irb) {}
+        IRB.irb_exit
+      rescue UncaughtThrowError
+        Kernel.exit
+      end
```

Comment:
This part is not needed.

`exit` command has this rescue part. It is a workaround for this bug https://bugs.ruby-lang.org/issues/18234.
Root cause of the bug is https://github.com/ruby/irb/issues/592.
Unlike `exit` command and `Kernel.exit` method, `disable_irb` method is not defined in Kernel.

Reply:
Thanks for the review!
I removed the unnecessary code https://github.com/ruby/irb/pull/898/commits/6724bf42a2b32223a0d67bec4fc0fcb5962aed2f